ginkgo-env

```diff
@@ -0,0 +1 @@
+export OPENEDX_RELEASE='GINKGO'
```

Comment:
```suggestion
```

Since tox already have that variable.

Reply:
I use it for devsite ginkgo (and I ought to add instructions for running devsite in "Ginkgo mode".  When I update the `Makefile` to act as a full wrapper around devsite for Ginkgo, then I'll delete this file